zebra-network/src/protocol/internal/response.rs

```diff
@@ -11,7 +11,7 @@ use std::{fmt, sync::Arc};
 use proptest_derive::Arbitrary;
 
 /// A response to a network request, represented in internal format.
-#[derive(Clone, Debug)]
+#[derive(Clone, Debug, Eq, PartialEq)]
```

Comment:
This was added so that the property test can do `prop_assert_eq!(response, Response::...)`.